test/platform/test-device-manager.ts

```diff
@@ -20,7 +20,10 @@ const COMMAND_REQUEST = createUdpDeviceCommand(
  * Returns a simple execute request used for testing
  * @param command A command to set in the sample request
```

Comment:
Should add a `@param params` here and a `@returns`.

Reply:
Done.